### ISSUE TITLE
[Bug]: Won't start on Mac

### ISSUE THREAD

**Issue by jjerome00** [httptoolkit/httptoolkit#673]
### Has this been reported before?

- [X] I have checked for existing reports of this issue

### Repro steps

1. Open Desktop App
2. ????

### How often does this bug happen?

Every time

### The desktop OS you're using

MacOS Sonoma 14.6.1; M1 Pro

### Details of other apps/devices

Version: 1.19.1
* Originally installed via HomeBrew 
* Also tried downloading from website (HttpToolkit-1.19.1.dmg) - same result
* Sometimes the app will display the broken screen, like in the screenshot
* Other times it will show no window at all - only the menu on my Mac

### Error screenshot

![http-toolkit-failure](https://github.com/user-attachments/assets/b09b55e4-abf8-4e71-a625-c3e4a147ff4a)

### Any other info?

* Tried uninstalling / reinstalling - same result
* Tried HomeBrew and version downloaded from website - same result
* I'm a subscribing/paying member, I can't afford to not have this not work

**Comment by jjerome00** [comment]
I had installers available for versions `1.19.0` and `1.14.7` and they seem to have the same problem. I'm wondering if my job's vpn/scaler is blocking a port or something? The only evidence against this is that I have coworkers running the app without issue.

**Comment by jjerome00** [comment]
TLDR I got it to work!
* I use zscaler while working from home, but I went into the office today and it loaded properly
* I've since gone home and tested again - and it is still working
    * I apologize but I do not know if the app had been updated before I started using it yesterday

As for you questions:
* Time was Dec 11th, 2024, around 10:30am PST & throughout the day
* The ip address is 198 . 151 . 49 . 81, although that probably has changed by now

**Comment by pimterry** [comment]
> TLDR I got it to work!

Great! Glad that's sorted for you now. 

> I've since gone home and tested again - and it is still working 

This doesn't necessarily mean that the issue is gone - once you load the first time, everything is cached and the app should work offline indefinitely (you only need working connectivity for the first time). There might still be issues here, but that will just mean your app won't update to new versions until you're on a connection where it can do so (probably not a big deal). You might be able to see those in the developer console if so.

> Time was Dec 11th, 2024, around 10:30am PST & throughout the day

Thanks for those details - unfortunately there's nothing obvious in my logs related to this.

My best guess is some kind of interaction between Zscaler and my CDN, with unusual traffic patterns from that resulted in an automatic block or similar somewhere, since the original 429 error listed here is a rate limit.

I'll keep an eye out for more reports and issues around that, and I can investigate further if this is a common reproducible issue, but if it's working for you and you don't see more problems coming up later then I think we're good for now.

**Comment by jjerome00** [comment]
Thanks for the diligence. It's appreciated. I love the product and I pay for it myself. If it turns out it's zscaler I may be able to get my company to make some adjustments.
